commons/src/main/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+quarkus.micrometer.enabled=true
```

Comment:
Do we need this file? I figure it will be overridden by the modules importing `commons` anyway.

Reply:
done